src/test/java/org/mongojack/TestQuerySerialization.java

```diff
@@ -56,17 +56,25 @@ public void setUp() {
     public void testSimpleEquals() {
         coll.save(new MockObject());
         String id = coll.findOne().id;
+        // with DBCursor
+        assertNotNull(coll.find().is("_id", id).next());
+        // with DBQuery
         assertNotNull(coll.findOne(DBQuery.is("_id", id)));
     }
 
     @Test
     public void testIn() {
         coll.save(new MockObject());
         String id = coll.findOne().id;
+        // with DBCursor
         assertThat(
                 coll.find()
                         .in("_id", id, new org.bson.types.ObjectId().toString())
                         .toArray(), hasSize(1));
+        // with DBQuery
+        assertThat(
+                coll.find(DBQuery.in("_id", id, new org.bson.types.ObjectId().toString()))
+                        .toArray(), hasSize(1));
     }
 
     @Test @Ignore // This test needs to be fixed
```

Comment:
can this `@Ignore` be removed? in https://github.com/mongojack/mongojack/pull/144 you added a comment that this PR fixes it

Reply:
Good catch! :-)
This probably survived when merging.